client/mass/charts.js

```diff
@@ -199,12 +205,6 @@ function getChartTypeList(self, state) {
 			}
 		},
 
-		{
```

Comment:
Not sure why this has to be deleted

Reply:
I talked about this with Xin, the same plot can be accessed through the dictionary button